elastic/kibana
# [ResponseOps] Revisit connector action retry back-off

Resolves https://github.com/elastic/kibana/issues/172518

## Summary

Updates the retry delay calculation to cap the delay at 1hr and introduces jitter. 

### Checklist



### To verify
- Create a rule and then force a retry failure
- Verify that the retry follows the pattern below:

  Attempt 1: now
  Attempt 2: 30s after the first attempt
  Attempt 3: 0 - 5m after the second attempt
  Attempt 4: 0 - 10m after the third attempt
  Attempt 5: 0 - 20m after the fourth attempt
  Attempt 6: 0 - 40m after the fifth attempt
  Attempt n: 0 - 1hr for all other attempts
